akka-cluster-sharding/src/main/scala/akka/cluster/sharding/Shard.scala

```diff
@@ -321,6 +339,49 @@ private[akka] class Shard(
   }
 }
 
+private[akka] object RememberEntityStarter {
+  def props(typeName: String, ids: Set[ShardRegion.EntityId]) = Props(new RememberEntityStarter(typeName, ids))
+}
+
+/**
+ * INTERNAL API: Actor responsible for starting entities when rememberEntities is enabled
+ */
+private[akka] class RememberEntityStarter(typeName: String, ids: Set[ShardRegion.EntityId]) extends Actor {
+
+  import context.dispatcher
+  import scala.concurrent.duration._
+
+  case object Tick
+
+  // TODO make configurable? more sane default?
+  val resendInterval = 5.seconds
+
+  val region = ClusterSharding(context.system).shardRegion(typeName)
+  var waitingForAck = ids
+
+  sendStart(ids)
+
+  val cancelable = context.system.scheduler.schedule(resendInterval, resendInterval, self, Tick)
+
+  def sendStart(ids: Set[ShardRegion.EntityId]): Unit = {
+    ids.foreach(id ⇒ region ! Shard.StartEntity(id))
+  }
+
+  override def receive = {
+    case Shard.StartEntityAck(id) ⇒
+      waitingForAck -= id
+      if (waitingForAck.isEmpty) context.stop(self)
```

Comment:
we should also remove them from the durable state, if they were started elsewhere. The `StartEntityAck` could carry some more info of where it was started